src/Telemetry/Samples/CSharp/Program.cs

```diff
@@ -0,0 +1,209 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Microsoft.Quantum.Telemetry.Samples.CSharp
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var stopwatch = Stopwatch.StartNew();
+
+            var telemetryConfig = new TelemetryManagerConfig()

```

Comment:
initializing and enabling telemetry should be only be enabled the the `TELEMETRY` compiler constant is on.

Reply:
Do we really need this `TELEMETRY` compiler constant?
Why not always build with telemetry included and opt-out during runtime if needed?

Reply:
Accepted you suggestion as discussed.